src/controllers/users.js

```diff
@@ -31,48 +78,26 @@ export default class usersController {
         email,
         password: hashedPassword,
         passport: passportNumber,
-        role: 'requester'
       });
-      const token = generateToken(
-        {
-          id: newUser.id,
-          email: newUser.email,
-          firstName,
-          lastName,
-          role: newUser.role
-        },
-        process.env.SECRETKEY
-      );
 
-      localStorage.setItem('token', token);
+      const token = generateToken({
+        id: newUser.id,
+        email: newUser.email,
+        firstName: newUser.firstName,
+        lastName: newUser.lastName
+      }, process.env.SECRETKEY);
 
-      return res
-        .status(201)
-        .json({ message: 'user successfully created', token });
+      localStorage.setItem('token', token);
+      return res.status(201).json({ message: 'Your account is successfully created.', token });
```

Comment:
I think to return token on verification processes is not necessary you can display on message

Reply:
@EugeneMunya , thanks for your feedback. It's solved now.